src/command_modules/azure-cli-keyvault/azure/cli/command_modules/keyvault/commands.py

```diff
@@ -108,3 +108,9 @@ def keyvault_update_setter(client, resource_group_name, vault_name, parameters):
 cli_keyvault_data_plane_command('keyvault certificate issuer admin list', custom_path.format('list_certificate_issuer_admins'))
 cli_keyvault_data_plane_command('keyvault certificate issuer admin add', custom_path.format('add_certificate_issuer_admin'))
 cli_keyvault_data_plane_command('keyvault certificate issuer admin delete', custom_path.format('delete_certificate_issuer_admin'))
+
+# default policy document
+cli_keyvault_data_plane_command('keyvault certificate get-default-policy', custom_path.format('get_default_policy'))
+
+# secret vm formatter
+cli_command(__name__, 'keyvault secret vm-format', custom_path.format('get_vm_format_secret'), factory)
```

Comment:
@johanste you and @tjprescott get to talk this over. I don't feel strongly where this lives, but I do feel that this will make secrets easier for users of the CLI.